### PR TITLE
Fix freezer goroutine leak

### PR DESCRIPTION
**Pull request by JekaMas**

### CODE REVIEW

File: core/rawdb/freezer.go [hunk 1]
```diff
@@ -391,3 +414,16 @@ func (f *freezer) repair() error {
 	atomic.StoreUint64(&f.frozen, min)
 	return nil
 }
+
+func (f *freezer) sleep(d time.Duration) error {
+	t := time.NewTimer(freezerRecheckInterval)
```
Comment (yazzaoui) [inline core/rawdb/freezer.go]: Shouldn't it be `t := time.NewTimer(d)` ?

File: core/rawdb/freezer.go [hunk 2]
```diff
@@ -254,122 +257,142 @@ func (f *freezer) Sync() error {
 func (f *freezer) freeze(db ethdb.KeyValueStore) {
 	nfdb := &nofreezedb{KeyValueStore: db}
 
+	ticker := time.NewTicker(freezerRecheckInterval)
+	defer ticker.Stop()
+	<-ticker.C
```
Comment (yazzaoui) [inline core/rawdb/freezer.go]: could you explain why waiting here was needed?

File: core/rawdb/freezer.go (hunk 1 above)
Comment (JekaMas) [reply core/rawdb/freezer.go]: You're right. Will fix.

File: core/rawdb/freezer.go (hunk 2 above)
Comment (JekaMas) [reply core/rawdb/freezer.go]: Its' the same code as it was, but I've rewrote `time.Sleep` to `time.NewTicker`

**Comment by JekaMas** [comment]
Closed in favor of https://github.com/clearmatics/autonity/pull/305

**Pull request closed without merging**